### PR TITLE
dns-rfc2136: add test coverage for PR #9672

### PR DESCRIPTION
**Pull request by alexzorin**
I noticed slightly too late that the unit tests were not updated during #9672. Here is some test coverage for the changes.

### CODE REVIEW

File: certbot-dns-rfc2136/certbot_dns_rfc2136/_internal/tests/dns_rfc2136_test.py [hunk 1]
```diff
@@ -40,8 +40,19 @@ def setUp(self):
 
         self.mock_client = mock.MagicMock()
         # _get_rfc2136_client | pylint: disable=protected-access
+        self.orig_get_client = self.auth._get_rfc2136_client
         self.auth._get_rfc2136_client = mock.MagicMock(return_value=self.mock_client)
 
+    def test_get_client(self):
```
Comment (ohemorange) [inline certbot-dns-rfc2136/certbot_dns_rfc2136/_internal/tests/dns_rfc2136_test.py:46]:
```suggestion
    def test_get_client_default_conf_values(self):
```
Given the comment, perhaps a more descriptive name then?

File: certbot-dns-rfc2136/certbot_dns_rfc2136/_internal/tests/dns_rfc2136_test.py [hunk 2]
```diff
@@ -100,7 +111,7 @@ def setUp(self):
         from certbot_dns_rfc2136._internal.dns_rfc2136 import _RFC2136Client
 
         self.rfc2136_client = _RFC2136Client(SERVER, PORT, NAME, SECRET, dns.tsig.HMAC_MD5,
-        TIMEOUT)
+        False, TIMEOUT)
```
Comment (ohemorange) [inline certbot-dns-rfc2136/certbot_dns_rfc2136/_internal/tests/dns_rfc2136_test.py:114]: We're changing the public signature? Are we holding this PR over to a major version? Are DNS plugins not covered by our external interface commitments?

**Review by ohemorange — CHANGES_REQUESTED** [review]

File: certbot-dns-rfc2136/certbot_dns_rfc2136/_internal/tests/dns_rfc2136_test.py (hunk 2 above)
Comment (alexzorin) [reply certbot-dns-rfc2136/certbot_dns_rfc2136/_internal/tests/dns_rfc2136_test.py:114]: We're in the `_internal` directory so I think we don't care.

Comment (ohemorange) [reply certbot-dns-rfc2136/certbot_dns_rfc2136/_internal/tests/dns_rfc2136_test.py:114]: 🤦 Good point.

**Review by ohemorange — APPROVED** [review]

**Pull request merged**